lib/cli/test/snapshots/vue/package.json

```diff
@@ -38,6 +38,6 @@
     "@storybook/addon-actions": "^4.0.0-alpha.12",
     "@storybook/addon-links": "^4.0.0-alpha.12",
     "@storybook/addons": "^4.0.0-alpha.12",
-    "babel-preset-vue": "^2.0.2"
```

Comment:
@Hypnosphi this looks suspicious?

Reply:
I add it inside app, to be consistent with `@storybook/react`:

https://github.com/storybooks/storybook/pull/3746/files#diff-48d46ebd591f6220a5142b66a694395b